src/main/java/com/amazonaws/encryptionsdk/model/DecryptionMaterials.java

```diff
@@ -12,7 +13,11 @@ public final class DecryptionMaterials {
   private DecryptionMaterials(Builder b) {
     dataKey = b.getDataKey();
     trailingSignatureKey = b.getTrailingSignatureKey();
-    encryptionContext = b.getEncryptionContext();
+    if (b.getEncryptionContext() != null) {
+      encryptionContext = b.getEncryptionContext();
+    } else {
+      encryptionContext = Collections.emptyMap();
+    }
```

Comment:
I wrote then and then though more. I don't know which one I like more. So I leave it up to the group.

```suggestion
    encryptionContext = b.getEncryptionContext();
    if (encryptionContext == null) {
      encryptionContext = Collections.emptyMap();
    }
```

Reply:
I like this more

Reply:
Ah, never mind. encryptionContext is final. The original one has to stay.